Cargo.toml

```diff
@@ -52,12 +52,14 @@ optional = true
 features = ["rt", "stm32l4x2", "stm32-usbd"]
 
 [features]
+dma = []
```

Comment:
Why exactly do we need a DMA feature flag? As far as I can tell DMA is supported in R-series MCUs. Is it just implementation that is not present/verified